src/chart/bar/BarSeries.js

```diff
@@ -57,6 +57,19 @@ export default BaseBarSeries.extend({
 
         // If use caps on two sides of bars
         // Only available on tangential polar bar
-        roundCap: false
+        roundCap: false,
+
+        backgroundStyle: {
```

Comment:
I'd suggest adding `showBackground` property for the `backgroundStyle`. And only creating background elements when `showBackground` is set true.

And the default `color` can be light gray.